model/src/main/proto/feedback_reporting.proto

```diff
@@ -0,0 +1,199 @@
+  string country_locale = 3;
+  // The ISO-3166 code for the language locale of where the report is sent fromm.
+  string language_locale = 4;
+}
+
+// Information collected about the user's device build.
+message FeedbackReportingDeviceContext {
+  // The model code of the user's device.
+  string device_model = 1;
+  // The version code for the SDK on the device.
+  int32 sdk_version = 2;
+  // The consumer brand of the device.
+  string device_brand = 3;
+  // The unique fingerprint ID for this specific build.
+  string build_fingerprint = 4;
+  // The device's telephony network type.
+  string phone_type = 5;
+  // The type of internet network connection the device is on.
+  enum NetworkType {
+    // Indicates an unspecified or unknown network type.
+    NETWORK_TYPE_UNSPECIFIED = 0;
+    // Indicates the device is connected to a wifi network.
+    WIFI = 1;
+    // Indicates that device is connected to a cellular network.
+    CELLULAR = 2;
+  }
+  NetworkType network_type = 6;
+  // The ISO-3166 code for the language the device is set to.
+  string device_language = 7;
+}
+
+// Information collected about the specific Oppia Android app.
+message FeedbackReportingAppContext {
+  // The entry point that feedback reporting was launched from.
+  enum EntryPoint {
+    // Indicates that the user started the feedback reporting feature from an unknown location.
+    ENTRY_POINT_UNSPECIFIED = 0;
+    // Indicates that the user started feedback reporting from the main hamburger menu.
+    NAVIGATION_DRAWER = 2;
+    // Indicates that the user started feedback reporting from the exploration player.
+    EXPLORATION_PLAYER = 3;
+    // Indicates that the user started feedback reporting from a revision card / session.
+    REVISION_CARD = 4;
+    // Indicates that the user started feedback reporting from a crash / the crash reporting dialog.
+    CRASH = 5;
+  }
+  EntryPoint entry_point = 1;
+  // Indicates the progress of a status based on the chapter progress for the topic.
+  enum TopicProgressStatus {
+    // The completion status is unknown.
+    STATUS_UNSPECIFIED = 0;
+    // No stories have been started in this topic.
+    NOT_STARTED = 1;
+    // The topic has been started but not completed by the player.
+    STARTED_NOT_COMPLETED = 2;
+    // The topic has been completed by the player.
+    COMPLETED = 3;
+  }
+  // Maps topic ID to the completion state of topic, based on state of stories in topics. Helps
+  // contextualize the user's app state and understand the scenario the report is filed during.
+  map<string, TopicProgressStatus> topic_progress = 2;
+  // The text size the app is set to.
+  ReadingTextSize text_size = 3;
+  // The text language the app is set to.
+  AppLanguage text_language = 4;
+  // The audio language the app is set to.
+  AudioLanguage audio_language = 5;
+  // The app's specific device settings.
+  DeviceSettings device_settings = 6;
+  // If the account sending the report is an admin account.
```

Comment:
```suggestion
  // Whether the account sending the report is the admin account.
```

Something about the current wording confused me.

Reply:
Done